.travis.yml

```diff
@@ -98,13 +98,16 @@ script:
       gcc -v;
 
       if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-        export QTEST_FUNCTION_TIMEOUT=1200000
```

Comment:
@saifulbkhan there's a 30 min lag in test because of mass slicing changes?

Reply:
Our new slicing method is _that_ much slower.

Reply:
As discussed offline, let's restrict the RT range for slicing to reduce the time or use other sample files.